client/public/css/home.css

```diff
@@ -169,11 +174,11 @@ New color scheme reference is in utils.css
     width: 100vw !important;
   }
 
-  .subtext {
+  /* .subtext {
```

Comment:
please don't comment things which aren't needed anymore, just delete them :)

Reply:
My bad lol I thought I got them all. Deleting the comment now.